wpeverest/user-registration
# UR-350 Fix - Field option value dynamically not rendered in form builder

### All Submissions:






### Changes proposed in this Pull Request:

This PR fixes the issue in which the updated field option value was not being rendered while editing form field settings.

Closes # .

### How to test the changes in this Pull Request:

1. Go to Form Settings.
2. Change default settings for select fields.
3. Check if the changes are being reflected without saving the form.

### Types of changes:




### Other information:




### Changelog entry

> Enter a short summary of all changes on this Pull Request. This will appear in the changelog if accepted.
